packages/api/src/sequelize/migrations/2023-11-13_00_create-cq-directory.ts

```diff
@@ -0,0 +1,63 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import * as shared from "../migrations-shared";
+
+// Use 'Promise.all' when changes are independent of each other
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async transaction => {
+    await shared.createTable(
+      queryInterface,
+      "cq_directory",
```

Comment:
nit: could have a const on line 4 and reference it here and on the `down` function.